### PR TITLE
add CollectionList component to to Investment Profiles list view

### PR DESCRIPTION
**Pull request by debitan**
## Description of change

Show the profiles tab within Investments and display a list of profiles using the `CollectionList` component.

You need to go to `/investments/profiles` to view the new collection. Showing the local nav tab on the projects screen will be in a following PR. 
 
### Before
FireShot Capture 011 - Investments - DIT Data Hub - localhost

### After
profiles collectionList

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to develop?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/investments/client/LargeCapitalProfileCollection.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    event.target.blur()
+    event.preventDefault()
+  }
+
+  const getPageUrl = (page) => `?page${page}`
+
+  useEffect(() => {
+    async function fetchData () {
+      const endpoint = `/investments/profiles/data/`
+      const { data } = await axios.get(endpoint, {
```
Comment (ian-leggett) [inline src/apps/investments/client/LargeCapitalProfileCollection.jsx]: Should we guard against no connection, If we can't get a response we should tell the user.

Comment (debitan) [reply src/apps/investments/client/LargeCapitalProfileCollection.jsx]: The component, as it is currently, does't provide for showing an error message - instead the spinner will just keep spinning as `isLoading` will stay set to true.
Maybe something for us to consider for a further iteration though?

Comment (web-bert) [reply src/apps/investments/client/LargeCapitalProfileCollection.jsx]: Is that really acceptable?

Comment (debitan) [reply src/apps/investments/client/LargeCapitalProfileCollection.jsx]: @web-bert Nope. And Ian's shown me how to do it without changing my component. Using the ErrorSummary from gov-uk react. Code is now updated.

File: src/apps/investments/client/LargeCapitalProfileCollection.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+const LargeCapitalProfileCollection = () => {
+  const [profiles, setProfiles] = useState([])
+  const [totalItems, setTotalItems] = useState(0)
+  const [isLoading, setIsLoading] = useState(true)
+  const [errorMessage, setErrorMessage] = useState(null)
+
+  const activePage = parseInt(useSearchParam('page') || 1, 10)
+  const setActivePage = (page) =>
+    window.history.pushState({}, '', `${window.location.pathname}?page=${page}`)
```
Comment (rafenden) [inline src/apps/investments/client/LargeCapitalProfileCollection.jsx]: We could reuse the `getPageUrl` function here, e.g.

```
const getPageUrl = (page) => `${window.location.pathname}?page=${page}`
```
and then

```
window.history.pushState({}, '', getPageUrl(page))
```

File: src/apps/investments/transformers/profiles.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  created_on,
+}) {
+  return {
+    headingText: investor_company.name,
+    headingUrl: `/companies/${investor_company.id}`,
+    itemId: investor_company.id,
+    metadata: [
+      {
+        label: `Updated on`,
+        value: moment(created_on).format('D MMMM YYYY'),
```
Comment (rafenden) [inline src/apps/investments/transformers/profiles.js]: We should probably store and reuse date formats as constants.

**Pull request merged**